packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -75,6 +81,8 @@ function ListViewBlockSelectButton(
 	const images = useListViewImages( { clientId, isExpanded } );
 	const { rootClientId } = useListViewContext();
 
+	const isConnected = getBlockAttributes( clientId )?.metadata?.bindings;
```

Comment:
Same as above, it can be now accessed through the block edit context.

Reply:
Similar to [this comment](https://github.com/WordPress/gutenberg/pull/59185#discussion_r1505200130), I'm not sure the context can be readily passed to the list view component, though I may be misunderstanding here.

Reply:
It's me who misunderstood how the components are composed 😄